src/main/java/net/coderbot/iris/pipeline/newshader/OptifineCorePatcher.java

```diff
@@ -0,0 +1,37 @@
+package net.coderbot.iris.pipeline.newshader;
+
+import net.coderbot.iris.gl.blending.AlphaTest;
+import net.coderbot.iris.shaderpack.transform.StringTransformations;
+import net.coderbot.iris.shaderpack.transform.Transformations;
+
+public class OptifineCorePatcher {
```

Comment:
Would like for this to be pulled out of this PR, ideally we'd hold off on this until we implement it in the rewritten patching system based on glsl-transformer.

Reply:
👍

Reply:
Done.